### ISSUE TITLE
Links to child pages in <a href=> are incorrect

### ISSUE THREAD

**Issue by mberkowski** [UMNLibraries/umedia#77]
Child page `<a href=>` links are incorrect, using only the item number rather than the full UMedia ID with its p16022coll9999 prefix.

For example https://umedia.lib.umn.edu/item/p16022coll340:8436 the child pages hrefs are like `https://umedia.lib.umn.edu/item/p16022coll340:8436/8415` when the code actually supports `https://umedia.lib.umn.edu/item/p16022coll340:8436/p16022coll340:8415`. The former cause 500 errors to render in the main page body where the item viewer should show if the URL is hit directly. Under normal circumstances, a JS action loads child pages into the viewer with their correct identifiers and this problem isn't visible.

But those broken links are also ending up in search engines while we do not want search engines to spider child pages at all.

Solutions:
1) Correct the views to generate child page links like `/item/p16022coll340:8436/p16022coll340:8415` instead of `.../8415` 
2) Put a rule in robots.txt not to spider `/item/p16022coll*/*` if that won't also prevent spidering the items in general (a future sitemap will help this)
3) Try and get the old ones out of search engines, maybe have Apache issue `410 Gone`.

**Comment by mberkowski** [comment]
Calls to `link_to` in app/views/child_searches/_page.html.erb and app/views/children/_page.html.erb should reference the child with its collection as `child_id: "#{child.collection}:#{child.id}"` instead of using only the id as `child_id: child.id`

```
<%= link_to item_path(id: child.field_parent_id.value, child_id: "#{child.collection}:#{child.id}"),
                  'data-action': 'viewer#select',
                  'data-id': child.field_parent_id.value,
                  'data-child_id': "#{child.collection}:#{child.id}",
```

**Comment by mberkowski** [comment]
Fixed in 0b08917f9a7124e3d515a457727377032b3ec0b4 along with `rel=nofollow` and new robots rules
